### PR TITLE
Fixes #656 and added spanish translation

### LINKED ISSUE #656: require-tree has wrong argument (Cannot read property 'findById' of undefined)
Since I've updated from 0.4.2 to 0.4.5, every time a user successfully login I get the following error:
```bash
/home/chat/lets-chat/app/controllers/presence.js:29
        User.findById(userId, function (err, user) {
            ^
TypeError: Cannot read property 'findById' of undefined
    at Namespace.<anonymous> (/opt/lets-chat/app/controllers/presence.js:29:13)
    at Namespace.emit (events.js:129:20)
    at Namespace.emit (/opt/lets-chat/node_modules/express.oi/node_modules/socket.io/lib/namespace.js:205:10)
    at /opt/lets-chat/node_modules/express.oi/node_modules/socket.io/lib/namespace.js:172:14
    at process._tickCallback (node.js:355:11)
```
Upon taking my afternoon inspecting code, I caught the **require-tree** package not loading any of the three (3) directories: models, middlewares and controllers.

After taking some minutes of research I decided to edit [app.js:33](https://github.com/sdelements/lets-chat/blob/master/app.js#L33-L35) as follows:
```javascript
    models = all(__dirname+'/app/models'),
    middlewares = all(__dirname+'/app/middlewares'),
    controllers = all(__dirname+'/app/controllers'),
```
Which apparently fixes this mess :)

Some info:
* ubuntu 15.04 64x
* nodejs v0.12.9
* npm v2.14.9
* mongo v2.6.3

Edit: wrong lines of code

### PR DESCRIPTION
**Pull request by yzaguirre**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ var _ = require('lodash'),
 var MongoStore = connectMongo(express.session),
     httpEnabled = settings.http && settings.http.enable,
     httpsEnabled = settings.https && settings.https.enable,
-    models = all('./app/models'),
-    middlewares = all('./app/middlewares'),
-    controllers = all('./app/controllers'),
+    models = all(__dirname+'/app/models'),
```
Comment (hhaidar) [inline app.js]: Do you mind using path.resolve for this?

**Comment by hhaidar** [comment]
thank you, just one minor thing

**Comment by yzaguirre** [comment]
I've made the switch to path.resolve

**Comment by hhaidar** [comment]
Thank you so much @yzaguirre

**Pull request merged**